### ISSUE TITLE
> ./start.sh start didn't find existing data 

### ISSUE THREAD

**Issue by joerk** [ct-Open-Source/ct-Smart-Home#2]
Hi, 

I did an ./start.sh update on an existing installation. 

Now with each ./start.sh start I get es new zigbee encryption key ... 

------cut 

pi@pi3:~/ct-Smart-Home $ ./start.sh start
data folder is missing. creating it
touch: 'data/mqtt/config/mosquitto.conf' kann nicht berührt werden: Keine Berechtigung
Zigbee2Mqtt configuration is missing. creating it.

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
This is your random Zigbee encryption key:

[0x22, 0x3d, 0x51, 0x25, 0x86, 0x3c, 0xc4, 0xf4, 0xe5, 0x47, 0xe1, 0x30, 0xfc, 0x14, 0xc2, 0x3c]

Store it safely or you will have to repair all of your devices.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by jvlker** [comment]
Hi @joerk, 
got the same problem. I solved it with changing line 122 (-f to -d)
```
if [ ! -d data ]; then
	build_data_structure    
fi
```

I'm not a pro in shell, but I think same thing is needed for line 79:
```
if [ ! -f data/zigbee/configuration.yaml ]; then
	create_zigbee2mqtt_config
fi
```

**Comment by merlinschumacher** [comment]
fixed with #10
